libraries/stdlib/samples/test/samples/collections/maps.kt

```diff
@@ -206,6 +206,34 @@ class Maps {
             assertPrints(totalValue(stats), "Total: [1787]")
         }
 
+        @Sample
+        fun getValueWithoutDefault() {
+            val map = mapOf(1 to "One", 2 to "Two")
+            assertFailsWith<NoSuchElementException> { map.getValue(3) }
+            assertPrints(map.getValue(1), "One")
+        }
+
+        @Sample
+        fun getValueWithDefault() {
+            val mapWithDefault = mapOf(1 to "One", 2 to "Two").withDefault { key ->
+                when {
+                    key == 0 -> "Zero"
+                    key > 0 -> "Positive"
+                    else -> "Negative"
+                }
+            }
+            val actual = (-1..3).associateBy( { it }, { mapWithDefault.getValue(it) } )
```

Comment:
More idiomatic to use `associateWith` in this case, e.g. `(-1..3).associateWith { mapWithDefault.getValue(it) }`

Reply:
done